### PR TITLE
Module buster-assertions is now called referee.

### PR DESCRIPTION
**Pull request by dkl-ppi**
@cjohansen, @jodal: Can either of you both please review my changes before we merge them into the busterjs/buster-docs repository?

### CODE REVIEW

File: modules/referee.rst [hunk 1]
```diff
@@ -1,25 +1,23 @@
 .. default-domain:: js
 .. highlight:: javascript
-.. _buster-assertions:
+.. _referee:
 
 =================
-buster-assertions
+referee
 =================
```
Comment (jodal) [inline modules/referee.rst]: The `===` rows above and below the header text should have the same length as the header text. I think longer may be allowed, but shorter is an actual reStructuredText error.

**Comment by jodal** [comment]
+1 with comments

File: modules/referee.rst (hunk 1 above)
Comment (dkl-ppi) [inline modules/referee.rst]: Fixed

**Comment by cjohansen** [comment]
:+1:

**Pull request merged**